### DISCUSSION TITLE
Form Summary

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#215] [Component suggestions]
<img width="480" alt="visuele weergave van het form summary component" src="https://github.com/nl-design-system/backlog/assets/545207/ac68bd56-0dfc-4785-8839-d0ce4895b758">

We willen graag meer leren over het **Form Summary** component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1706798708688949)

---

## Naam
Form Summary

## Doel
Overzicht van ingevoerde data in een formulier.

## Gerelateerde componenten
[Data Summary](https://github.com/orgs/nl-design-system/discussions/356), [Description List](https://github.com/orgs/nl-design-system/discussions/216), [Table](https://github.com/orgs/nl-design-system/discussions/294)

## Richtlijnen
[Meerdere stappen in een formulier - NL Design System](https://www.nldesignsystem.nl/richtlijnen/formulieren/meerdere-stappen)
Onze richtlijnen rondom formulieren die uit meerdere stappen bestaan.

## Onderzoek
[A live service is not the end of the story - gov.uk blog](https://dwpdigital.blog.gov.uk/2016/07/08/a-live-service-is-not-the-end-of-the-story/)
Blogpost van gov.uk over de Carer’s Allowance Service waar dit component is meegenomen tijdens onderzoek.

[Contactformulier voor WMEBV - Gebruikersonderzoeken.nl](https://gebruikersonderzoeken.nl/docs/onderzoek-bekijken/wmebv/vng-online-formulieren#informatie-controleren-en-versturen-3)
Onderzoek waarbij de Form Summary component een rol speelde op de 'Controleer uw gegevens' pagina.

**Comment by sergei-maertens** [comment]
## Open Formulieren

[Storybook](https://open-formulieren.github.io/open-forms-sdk/?path=/story/private-api-formstepsummary--default)

---

### Screenshots / Varianten

**Default**
![Scherm­afbeelding 2024-02-05 om 11 25 41](https://github.com/nl-design-system/backlog/assets/545207/55d026ef-6861-448d-b7f1-8a6b3cc27c9e)

**Not editable**
![Scherm­afbeelding 2024-02-05 om 11 23 19](https://github.com/nl-design-system/backlog/assets/545207/c8ff943f-b23e-41f8-93f8-de6d29ff3d47)

**With field
![Scherm­afbeelding 2024-02-05 om 11 24 02](https://github.com/nl-design-system/backlog/assets/545207/f105bcd3-da37-4dbb-a4bb-056e66006519)
set not as first element**

---

### Notities
* We groeperen de gegevens per formulierstap.
* Edit link gaat altijd naar de hele stap waar het veld voorkomt.
* Verschillende weergaven op desktop vs. mobile (mobile is vertically stacked).
* Invoergegevens kunnen gegroepeerd zijn in een veldengroep, kolommen of een item van een herhalende groep zijn, deze groepering komt ook terug in het overzicht.

**Comment by jeffreylauwers** [comment]
## Rotterdam

[Figma](https://www.figma.com/file/ZWSC4gCrOXRUR9UX3aoZ8x/Gemeente-Rotterdam-Design-System?type=design&node-id=971-6482&mode=design&t=YZJvTJ1NWt6OGGhd-4)

---

### Screenshots / Varianten

**In combinatie met Category title component**
![Scherm­afbeelding 2024-02-05 om 11 55 26](https://github.com/nl-design-system/backlog/assets/545207/8fb593b9-3a44-45c9-be60-1d1294bed7a3)

---

### Notities
- Niet specifiek een Summary list component, maar wel iets dat in de buurt komt in combinatie met het Category title component.
- Let wel; layout is gelijk aan combinatie van Form field label en form controls dus wellicht zit hier een ander idee achter (bijvoorbeeld read-only weergave)

**Comment by jeffreylauwers** [comment]
## Utrecht

[Bekijk ook de toevoeging vanuit Utrecht in de Data Summary discussion ](https://github.com/orgs/nl-design-system/discussions/356#discussioncomment-11163817)

**Comment by jeffreylauwers** [comment]
## DUO

[Bekijk ook de toevoeging vanuit DUO in de Data Summary discussion ](https://github.com/orgs/nl-design-system/discussions/356#discussioncomment-11163654)

**Comment by jeffreylauwers** [comment]
Tijdens de Open Dag van september de naamgeving van de componenten Summary List en Description List besproken. Uiteindelijk zijn we uitgekomen op Form Summary, [Data Summary](https://github.com/orgs/nl-design-system/discussions/356) en [Description List](https://github.com/orgs/nl-design-system/discussions/216).

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Form Summary op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=form+summary)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Form Summary op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=form+summary)

**Comment by bas-kroese** [comment]
Tijdens de NLDS Design Open Hour van 11 aug 2026 kwam dit onderwerp aan bod:

Bij Amsterdam hebben we ervoor gekozen om component '**Data Summary**' aan te bieden, waarmee direct ook het patroon form summary kan worden gemaakt. 

Vooralsnog alleen in Figma beschikbaar: https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/Amsterdam-Design-System?node-id=48983-138

Wanneer op een pagina een data summary moet worden gemaakt met meerdere groepen (bijvoorbeeld meerdere toegevoegde personen in een formulier) dan kiezen we er vooralsnog voor om het op deze manier op te lossen op de pagina:
<img width="713" height="1249" alt="image" src="https://github.com/user-attachments/assets/42e948a0-3ece-4edf-be4d-fa8a4724b20d" />

Redenering:

- Meer flexibiliteit in de opmaak (risico op te veel variatie in de uitwerking zou je kunnen afdekken met goede documentatie en patronen)
- Voor screenreaders ook prima in te richten. Dat word al lastiger waarschijnlijk als je het binnen een component wil afdekken.

**Reply by RubenSibon** [reply]
Now that we’re actually implementing the Data Summary (i.e. Data List), we come to the conclusion there isn’t much added value for a Form Summary component on top of that. It’s mostly the same thing and the stuff that is added can be a minor feature of Data Summary (the heading for example) or be solved in a composition with other components. See [my remarks in the discussion](https://github.com/orgs/nl-design-system/discussions/356#discussioncomment-18403329) on Data Summary.
